### PR TITLE
Allow installation with PHP8, add Symfony 5.2 to tests

### PR DESCRIPTION
**Pull request by ker0x**

### CODE REVIEW

**Review by jrushlow — APPROVED** [review]

**Comment by jrushlow** [comment]
thanks @ker0x ! we should get this in pretty quick!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
       fail-fast: false
       matrix:
         php-versions: ['7.2', '7.3', '7.4']
```
Comment (jrushlow) [inline .github/workflows/ci.yml:55]:
```suggestion
        php-versions: ['7.2', '7.3', '7.4', '8.0']
```

Comment (jrushlow) [reply .github/workflows/ci.yml:55]: Whoops! Just noticed we didn't have PHP 8 in our test matrix. We'll also need to do the same in other `php-verions` matrix in this file

**Comment by jrushlow** [comment]
Tests fail as the Ubunut 18.04 runner w/ PHP8 is not yet deployed - https://github.com/actions/virtual-environments/issues/2157 
I'll look into an alternative in the meantime that does not involve refactoring our entire CI. Off the top of my head this should not be a blocker for this PR.

**Comment by ker0x** [comment]
Should I update my PR and remove PHP 8.0 from the tests matrix ?

**Comment by weaverryan** [comment]
Thanks @ker0x!

Obviously, my vote was to keep 8.0 in the matrix. Hopefully we'll have a php 8 image in a few days and things will pass.

Cheers!

**Pull request merged**